extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/OpenTelemetryUtil.java

```diff
@@ -16,6 +20,7 @@ public final class OpenTelemetryUtil {
     public static final String SPAN_ID = "spanId";
     public static final String SAMPLED = "sampled";
     public static final String PARENT_ID = "parentId";
+    private static final Set<String> SPAN_DATA_KEYS = new HashSet<>(Arrays.asList(TRACE_ID, SPAN_ID, SAMPLED, PARENT_ID));
```

Comment:
```suggestion
    private static final Set<String> SPAN_DATA_KEYS = Set.of(TRACE_ID, SPAN_ID, SAMPLED, PARENT_ID);
```
:slightly_smiling_face:

Reply:
ok